pkg/oss/errors.go

```diff
@@ -0,0 +1,15 @@
+package oss
+
+import (
+	"errors"
+	"fmt"
+)
+
+var ParamError = errors.New("Oss parameters error")
```

Comment:
```suggestion
var ParamError = errors.New("OSS parameters error")
```
It is strange. but maybe fine because the old version looks like this.